src/pages/add/copayers/copayers.html

```diff
@@ -0,0 +1,62 @@
+<ion-header>
+
+  <ion-navbar>
+    <ion-title *ngIf="wallet">{{ wallet.name }}</ion-title>
+  </ion-navbar>
+
+</ion-header>
+
+
+<ion-content>
+  <div *ngIf="wallet" [hidden]="wallet.notAuthorized">
+    <ion-list class="copayers-secret">
+      <ion-item translate>
```

Comment:
Translate directive should be used as pipe, not in ion labels.

Reply:
Got it. Thanks